boost.BUILD

```diff
@@ -996,6 +996,23 @@ boost_library(
     ],
 )
 
+boost_library(
+    name = "python",
+    exclude_src = ["**/fabscript"],  
```

Comment:
Why not `exclude_src = ["libs/python/src/fabscript"],`?

Also, nitpick, but there's trailing whitespace on this line